pkg/packages/manager.go

```diff
@@ -77,6 +78,32 @@ func processInstalled(mc *ManagerContext) bool {
 		mc.Package.Status.State = api.StateUpdating
 		mc.RequeueAfter = retryShort
 		return true
+	} else {
+
+		var err error
+		newValues := make(map[string]interface{})
+		err = yaml.Unmarshal([]byte(mc.Package.Spec.Config), &newValues)
+		if err != nil {
+			mc.Log.Error(err, "unmarshaling current package configuration")
+			mc.Package.Status.Detail = err.Error()
+			mc.RequeueAfter = retryShort
+			return true
+		}
+
+		needs, err := mc.PackageDriver.IsConfigChanged(mc.Ctx, mc.Package.Name,
+			newValues)
+		if err != nil {
+			mc.Log.Error(err, "checking necessity of reconfiguration")
+			mc.Package.Status.Detail = err.Error()
+			mc.RequeueAfter = retryShort
```

Comment:
This could probably be long or very long, we should be notified if they update/fix

Reply:
I re-read what I could find about this in kubebuilder, and am now thinking that `retryNever` should be used a lot more frequently than we are. Let me know if you disagree or understand something different.

Reply:
Event based is better than polling, so `retryNever` makes sense. I guess in the case of an error though, we might want to retry *sometime* in case the problem was ephemeral

Reply:
My understanding is that k8s will queue the resources periodically, even if not requested. Let me see if I can find the docs on that.

Reply:
I can't find a solid reference to that right now. Will set to verylong.